tasks/change-granule-collection-s3/src/types.ts

```diff
@@ -1,9 +1,10 @@
+import { ApiGatewayLambdaHttpProxyResponse } from '@cumulus/api-client/types';
 import { BucketsConfigObject } from '@cumulus/common/types';
 import { ApiFile, ApiGranuleRecord, CollectionRecord } from '@cumulus/types';
 export const MB = 1024 * 1024;
 
 type TestMethods = {
-  listGranulesMethod: (granuleIds: string[], collectionId: string) => ApiGranuleRecord,
+  listGranulesMethod: (params: Object) => Promise<ApiGatewayLambdaHttpProxyResponse>,
```

Comment:
This is fair as we're using a downstream validator.  I was going to object to Object being used here, but looking downstream there's a validator that makes that concern ... invalid.    Nice work.